{
  "deprecated": false,
  "cpeName": "cpe:2.3:a:ibm:planning_analytics_cloud:2.0.0:*:*:*:*:*:*:*",
  "cpeNameId": "9B831338-6D35-4470-9AF3-3545E663BD3E",
  "created": "2021-05-21T18:07:38.740",
  "lastModified": "2021-06-07T18:52:45.080",
  "titles": [
    {
      "title": "IBM Planning Analytics Cloud 2.0.0",
      "lang": "en"
    }
  ],
  "refs": [
    {
      "ref": "https://exchange.xforce.ibmcloud.com/vulnerabilities/186401",
      "type": "Advisory"
    },
    {
      "ref": "https://www.ibm.com/us-en?lnk=m?lnk=m",
      "type": "Vendor"
    }
  ]
}
